esphome/components/statsd/statsd.cpp

```diff
@@ -0,0 +1,156 @@
+#include "esphome/core/log.h"
+
+#include "statsd.h"
+
+namespace esphome {
+namespace statsd {
+
+// send UDP packet if we reach 1Kb packed size
+// this is needed since statsD does not support fragmented UDP packets
+static const uint16_t SEND_THRESHOLD = 1024;
+
+static const char *const TAG = "statsD";
+
+void StatsdComponent::setup() {
+#ifndef ESP8266
```

Comment:
All of our definitions are prefixed with `USE_` in esphome.
```suggestion
#ifndef USE_ESP8266
```

Reply:
intresting that ESP8266 works too, may from the Arduino framework.

changed.